neuror/unravel.py

```diff
@@ -92,6 +92,8 @@ def _unravel_section_path_length(sec, window_half_length, soma, legacy_behavior)
         _l_right = np.cumsum(intervals[min(n_points - 2, window_center):])
         window_start = np.argmin(abs(_l_left - window_half_length))
         window_end = np.argmin(abs(_l_right - window_half_length))
+        assert window_start == window_end, f"""we cannot unravel with path length,
```

Comment:
I am curious how is this check ensuring that points are not too far apart?

Reply:
if left and right side are the same, it means there is a single point, so the algorithm will fail. But don't worry, I'll clean this one up, I made it real quick to answer the issue raised.